bot.py

```diff
@@ -64,12 +64,36 @@ async def stopwach(ctx):
 
 
 @client.command(aliases=["hourly-reminder", "set-hourly-reminder"])
-async def hourly_reminder(ctx, task):
+async def hourly_reminder(ctx, *task):
     # Takes input from the user (task) about what they would like to accomplish
-    # Send a reminder after one hour to see how far they have come up with the task
-    # At the end of every hour, ask the user if they have completed the task.
-    # Wait for the stop command from user (You may use "client.wait_for")
-    pass
+    task = " ".join(task)
+    await ctx.send(ctx.author.mention + f" Task: {task}, saved successfully")
+
+    # Start reminder loop
+    reminder_loop.start(ctx, task)
```

Comment:
add a function to let the user stop the remainders and add a new reminder all together